platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/api/internal/LoggingSystem.java

```diff
@@ -120,9 +118,7 @@ public void addHandler(@NonNull final LogHandler handler) {
      * @param handler the handler to remove
      */
     public void removeHandler(@NonNull final LogHandler handler) {
-        if (handler == null) {
-            EMERGENCY_LOGGER.logNPE("handler");
-        } else {
+        if (!LoggerUtils.checkIsNull(handler, "handler")) {
```

Comment:
I can not find LoggerUtils. Looks like that is new but it is not part of the PR

Reply:
My bad, thanks for pointing it out